images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -37,6 +37,18 @@ RUN set -ex \
         swig \
         tk-dev \
         uuid-dev \
+        python3-dev \
+        libffi-dev \
+        gdb \
+        lcov \
+        pkg-config \
+        libgdbm-dev \
+        libgdbm-compat-dev \
+        liblzma-dev \
+        libreadline6-dev \
+        lzma \
+        lzma-dev \
+        zlib1g-dev \
     && rm -rf /var/lib/apt/lists/*
```

Comment:
Double confirming if we need to add these for other images as well considering the app services team only uses the github-actions image?

Reply:
This depends on whether we don't want to add the 99MB to the other images. If don't add them, we'll have to continue adding this case-by-case. Did you have concerns this would increase the Jamstack image download performance?

Reply:
This depends on whether we don't want to add the 99MB to the other images. If don't add them, we'll have to continue adding this case-by-case. Did you have concerns this would increase the other image download performance?

Reply:
Yep, I thought this may increase the size of the other images that App service does not consume but since it is only ~100 MB, it _should not_ increase the time to pull down the image significantly.